### ISSUE TITLE
How i put filters above or below column's names ?

### ISSUE THREAD

**Issue by germ1f** [ratiw/vuetable-2#49]
Hello,

I use vuetable-2 that I find very well, and i start with vuejs 2, i would like add my filters above or below column's names with html tag thread tr td. (like this grid example : http://grids-demo.herokuapp.com/demo/example4)

I have this  http://imgur.com/a/I9laA and it's work.

Thank you in advance for your help.

**Comment by ratiw** [comment]
@germ1f Have you check out this [tutorial lesson](https://github.com/ratiw/vuetable-2-tutorial/blob/master/doc/lesson-13.md)?

**Comment by germ1f** [comment]
Yes, i see your tutorial and it's work but i would like add my filter under colum's name in vuetable but i don't know if it's possible.

**Comment by ratiw** [comment]
Sorry, it's not possible. The only way right now is to fork it and try to implement that.

**Comment by germ1f** [comment]
ok, thanks ! I try this !

**Comment by SimplyCorey** [comment]
@ratiw Would you be open to a pull request for the column search fields?  I can go ahead and add them and show them by using  a prop.

**Comment by ratiw** [comment]
@SimplyCorey You are very welcome and a very nice of you to share it. 

However, please don't feel bad if it doesn't get merged. But I'll make it will be linked to when someone asks about this.

**Comment by antoarva** [comment]
I made an extended version of @germ1f  with simultaneously multiple column filter. Also added 3 types of filter a simple input, a datepicker or a dropdown menu which accepts dynamic data. Until you release your next major version I believe is a solution. @ratiw thank you for your implementation.

[Multiple column filters](https://github.com/ratiw/vuetable-2/pull/270)
